Robot Code/src/org/usfirst/frc/team3316/robot/commands/climbing/MoveDown.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc.team3316.robot.commands.climbing;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.commands.DBugCommand;
+
+public class MoveDown extends DBugCommand
+{
+
+	private double speed;
+	
+	public MoveDown() {
+		requires(Robot.climbing);
+	}
+	
+	protected void init()
+	{}
+
+	protected void execute()
+	{
+		speed = -(double) config.get("climbing_Speed");
```

Comment:
Pick a different decent speed. Since on this way the mechanism works differently (holding back the robot weight instead of pulling it).